semantics/executable-spec/small-steps.cabal

```diff
@@ -32,19 +32,24 @@ library
                      , Control.State.Transition.Simple
                      , Data.AbstractSize
                      , Data.Relation
+                     , Data.CannonicalMaps
+                     , Data.MemoBytes
+                     , Data.Coders
+                     , Data.SetAlgebraInternal
```

Comment:
Why `Data.SetAlgebraInternal` but `Control.SetAlgebra`? We should keep the same prefix...

Reply:
So now the actual code is in
Control/Iterate/SetAlgebra,hs
And the interface is in
Control/SetAlgebra.hs